src/components/dashboard/SendLinkSummary.js

```diff
@@ -292,17 +256,22 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
       showErrorDialog('Could not complete transaction. Please try again.')
       log.error('Something went wrong while trying to generate send link', e.message, e)
     }
-  }, [amount, reason, counterPartyDisplayName, survey, showErrorDialog, screenProps])
+  }, [amount, reason, counterPartyDisplayName, survey, showErrorDialog, goToRoot])
```

Comment:
why source haven't used here ? were there no code repeating issue ?

Reply:
Yeah, there is no such code duplicates. The values passed to deps are used separately in the scope of this fn.